wizeline/project-lab
# Feature/ Build generic select component

Feat - Build generic select component

#### What does this PR do?

Create a generic component to use as an input selector within a form and remove the repetitive code. In this way, the project will have a single component available that can be used on multiple pages.

#### Where should the reviewer start?

app / core / components / InputSelect / index.tsx
app / pages / create  / index.tsx

#### How should this be manually tested?

1. Pull the changes and run the project.
2. Go to the "Create a proposal" page and fill out the form.
4. Verify the following inputs behavior: Category selector and Project status selector.
3. Then, go to the "Project details" page and select the edit icon.
4. Change the selectors values in different cases to verify behaviors and the correct values updating.

#### What are the relevant tickets?

#137
